static/app/views/dashboardsV2/widgetCard/widgetCardChartContainer.tsx

```diff
@@ -70,55 +54,7 @@ export function WidgetCardChartContainer({
   expandNumbers,
   onDataFetched,
 }: Props) {
-  function issueTableResultComponent({
```

Comment:
took the opportunity that I was checking for re-renders and moved this to a new component called issueWidgetCard.